wp-modules/get-module-data/tests/PatternManagerGetModuleDataTest.php

```diff
@@ -24,7 +24,7 @@ public function test_module_dir_path() {
 	 * Gets the dir url.
 	 */
 	public function test_module_dir_url() {
-		$expected = 'http://localhost:1001/wp-content/plugins/pattern-manager/wp-modules/get-module-data/';
+		$expected = 'http://localhost:8080/wp-content/plugins/pattern-manager/wp-modules/get-module-data/';
```

Comment:
Ideally, this would assert that `$result` ends in `wp-content/plugins/pattern-manager/wp-modules/get-module-data/'`